src/app/flows/fines/fines-acc/fines-acc-debtor-add-amend/fines-acc-debtor-add-amend.component.ts

```diff
@@ -1,8 +1,34 @@
 import { Component } from '@angular/core';
+import { FinesAccDebtorAddAmendFormComponent } from './fines-acc-debtor-add-amend-form/fines-acc-debtor-add-amend-form.component';
+import { AbstractFormParentBaseComponent } from '@hmcts/opal-frontend-common/components/abstract/abstract-form-parent-base';
+import { IFinesAccDebtorAddAmendForm } from './interfaces/fines-acc-debtor-add-amend-form.interface';
+import { FINES_ACC_DEBTOR_ADD_AMEND_FORM } from './constants/fines-acc-debtor-add-amend-form.constant';
 
 @Component({
   selector: 'app-fines-acc-debtor-add-amend',
-  imports: [],
+  imports: [FinesAccDebtorAddAmendFormComponent],
   templateUrl: './fines-acc-debtor-add-amend.component.html',
 })
-export class FinesAccDebtorAddAmend {}
+export class FinesAccDebtorAddAmend extends AbstractFormParentBaseComponent {
+  protected readonly prefilledFormData: IFinesAccDebtorAddAmendForm =
+    this['activatedRoute'].snapshot.data['debtorAmendFormData'] || FINES_ACC_DEBTOR_ADD_AMEND_FORM;
```

Comment:
This is effectively being twice
```suggestion
  protected readonly prefilledFormData: IFinesAccDebtorAddAmendForm =
    this['activatedRoute'].snapshot.data['debtorAmendFormData']
```